tests/php/test-class-amp-meta-box.php

```diff
@@ -121,6 +121,11 @@ public function test_enqueue_admin_assets() {
 	 * @covers ::enqueue_block_assets()
 	 */
 	public function test_enqueue_block_assets() {
+		// Block validation script uses features only available beginning with WP 5.6.
+		if ( version_compare( get_bloginfo( 'version' ), '5.6', '<' ) ) {
+			$this->markTestSkipped( 'The block validation script is only available in WordPress 5.6 and above.' );
+		}
```

Comment:
Why is this needed now if previously we were testing back to 4.9?

Reply:
Because we are checking for dependency support in `enqueue_block_assets()`

https://github.com/ampproject/amp-wp/blob/956c8d298764e4b22b936347d7c8b591b2fadfc2/includes/admin/class-amp-post-meta-box.php#L220-L224